tools/lint-all

```diff
@@ -83,9 +83,9 @@ def check_pep8(files):
     failed = False
     ignored_rules = [
         'E402', 'E501', 'W503', 'E711', 'E201', 'E203', 'E202', 'E128', 'E226', 'E124',
-        'E126', 'E127', 'E121', 'E122', 'E123', 'E266', 'E265', 'E261', 'E221', 'E303',
+        'E126', 'E127', 'E121', 'E123', 'E266', 'E265', 'E261', 'E221', 'E303',
         'E241', 'E712', 'E702', 'E401', 'E115', 'E114', 'E111', 'E731', 'E302', 'E129',
-        'E741', 'E714', 'W391', 'E713', 'E502', 'E131', 'E305', 'E251', 'E306',
```

Comment:
You should remove 'E131' in the commit which takes care of E131 errors. This way linter tests will not fail for this commit.